tests/test_recipes.py

```diff
@@ -643,19 +645,19 @@ def test_no_replacement(self):
 
 
 class RandomCombinationWithReplacementTests(TestCase):
-    """Tests for ``random_combination_with_replacement()``"""
+    """Tests for ``random_combination_with_replacement()``."""
 
     def test_replacement(self):
-        """ensure that elements are sampled with replacement"""
+        """Ensure that elements are sampled with replacement."""
         items = range(5)
         combo = mi.random_combination_with_replacement(items, len(items) * 2)
         self.assertEqual(2 * len(items), len(combo))
         if len(set(combo)) == len(combo):
             raise AssertionError("Combination contained no duplicates")
 
     def test_pseudorandomness(self):
-        """ensure different subsets of the iterable get returned over many
-        samplings of random combinations"""
+        """Ensure different subsets of the iterable get returned over many
+        samplings of random combinations."""
```

Comment:
An artifact left from giving a try to the pydocstyle (D) rules. There is a strikingly great number of them throughout the source code and ignoring test files does not help out.